tests/upgrades/test_client.py

```diff
@@ -27,373 +28,303 @@
-    """
-    client_os = client_os.upper()
-    from_ver = settings.upgrade.from_version
-    rhel_prod_name = 'scenarios_rhel{}_prod'.format(client_os[-1])
-    rhel_repo_name = f'{rhel_prod_name}_repo'
-    rhel_url = settings.rhel7_os
-    if rhel_url is None:
-        raise ValueError(
-            'The RHEL Repo URL environment variable for OS {} '
-            'is not provided!'.format(client_os)
-        )
-    rhel_prod = entities.Product(name=rhel_prod_name, organization=org.id).create()
-    if sat_state.lower() == 'pre' and from_ver in ['6.1', '6.2']:
-        rhel_repo = entities.Repository(
-            name=rhel_repo_name, product=rhel_prod, url=rhel_url, content_type='yum'
-        ).create()
-    else:
-        rhel_repo = entities.Repository(
-            name=rhel_repo_name,
-            product=rhel_prod,
-            url=rhel_url,
-            content_type='yum',
-            verify_ssl_on_sync=False,
-        ).create()
-    call_entity_method_with_timeout(rhel_repo.sync, timeout=1400)
-    if sat_state.lower() == 'pre':
-        product_name = 'Red Hat Enterprise Linux Server'
-        repo_name = 'Red Hat Satellite Tools {} for RHEL {} Server RPMs x86_64'.format(
-            from_ver, client_os[-1]
-        )
-        tools_prod = entities.Product(organization=org.id).search(
-            query={'per_page': 1000, 'search': f'name="{product_name}"'}
-        )[0]
-        tools_repo = entities.Repository(organization=org.id, product=tools_prod).search(
-            query={'per_page': 1000, 'search': f'name="{repo_name}"'}
-        )[0]
-    elif sat_state.lower() == 'post':
-        product_name = 'scenarios_tools_product'
-        tools_repo_url = settings.sattools_repo[client_os.lower()]
-        if tools_repo_url is None:
-            raise ValueError(
-                'The Tools Repo URL environment variable for '
-                'OS {} is not provided!'.format(client_os.lower())
-            )
-        repo_name = f'{product_name}_repo'
-        tools_prod = entities.Product(organization=org.id).search(
-            query={'search': f'name={product_name}'}
-        )
-        if not tools_prod:
-            tools_prod = entities.Product(name=product_name, organization=org.id).create()
-            tools_repo = entities.Repository(
-                name=repo_name, product=tools_prod, url=tools_repo_url, content_type='yum'
-            ).create()
-            tools_repo.sync()
-        else:
-            tools_repo = entities.Repository(organization=org.id, product=tools_prod).search(
-                query={'search': f'name={repo_name}'}
-            )
-    tools_cv = entities.ContentView(
-        name=ak_name + '_cv', label=ak_name + '_cv', organization=org.id
+@pytest.fixture(scope='module')
+def pre_upgrade_repo(default_org):
+    """Enable custom errata repository"""
+    pre_upgrade_repo = entities.Repository(
+        url=FAKE_0_YUM_REPO, product=entities.Product(organization=default_org).create()
```

Comment:
```suggestion
        url=FAKE_0_YUM_REPO, product=module_product
```
Use module_product global fixture

Reply:
OK, done that